### DISCUSSION TITLE
S3 getting software.amazon.awssdk.core.exception.SdkClientException: Unable to execute HTTP request: The target server failed to respond

### DISCUSSION THREAD

**Discussion by ShashankGupta28** [aws/aws-sdk-java-v2#3397] [General]
Hi, i am facing

 `software.amazon.awssdk.core.exception.SdkClientException: Unable to execute HTTP request: The target server failed to respond`

can anyone suggest me what wrong i am doing

https://stackoverflow.com/questions/73546820/aws-sdk-java-s3-getting-software-amazon-awssdk-core-exception-sdkclientexception

**Comment by yasminetalby** [comment]
Hello @ShashankGupta28 ,

Thank you very much for your message. 
This error is potentially caused by closing getObject() mid-request.
Can you you please provide [wire logs](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/logging-slf4j.html) associated with this behavior?

I have noticed that you have mentioned that you are closing getObject() because you were experiencing `Unable to execute HTTP request: Timeout waiting for connection from pool`
Have you attempted to increase the connection timeout via [ClientConfiguration](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/ClientConfiguration.html)?



Best,

Yasmine
